opencl-aot/source/main.cpp

```diff
@@ -307,11 +344,37 @@ int main(int Argc, char *Argv[]) {
                               "OpenCL ahead-of-time (AOT) compilation tool");
 
   // step 1: perform checks for command line options
-  bool IsFileExists = false;
-  sys::fs::is_regular_file(OptInputBinary, IsFileExists);
-  if (!IsFileExists) {
-    std::cerr << "File " << OptInputBinary << " does not exist!" << '\n';
-    return OPENCL_AOT_FILE_NOT_EXIST;
+
+  // Setup default name of output file.
+  std::string OutputFileName((OptDevice == fpga_fast_emu) ? "aot-output.aocx"
+                                                          : "aot-output.bin");
+  if (OptOutputElf.getNumOccurrences())
+    OutputFileName = OptOutputElf;
+
+  // Get input files
+  if (!OptInputFile.getNumOccurrences()) {
+    std::cerr << "no input files";
+    return OPENCL_AOT_LIST_OF_INPUT_BINARIES_IS_EMPTY;
+  }
+
+  std::vector<std::string> InputFileNames;
+  // Split input files at ','s and ignore empty file name.
```

Comment:
Multiple input files are passed using space as a delimeter in LLVM-based tools, like
clang source1.cpp source2.cpp
Can you please update `cl::opt<std::string> OptInputFile` (and help message) to support multiple files?